### PR TITLE
add _candidates_en/YoshitakaOkuda_3.md

### LINKED ISSUE #80: 英訳: 奥田 佳享 さんによる「Reactive Messaging Patternsに学ぶシステム間統合」
ref: #74

### PR DESCRIPTION
**Pull request by yohei1126**
fix #80
ref. #74

### CODE REVIEW

**Comment by yohei1126** [comment]
@smroy Could you check?

File: _candidates_en/YoshitakaOkuda_3.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+  - Best Practices
+  - Software Design and Architecture
+  - Microservices
+suggestions:
+  - People who want to read Reactive Messaging Patterns
+  - People who are interested in systems integrations using Akka
+  - People who want to know how to apply Akka
+---
+The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP） was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.
```
Comment (smroy) [inline _candidates_en/YoshitakaOkuda_3.md]: +The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP）was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.

File: _candidates_en/YoshitakaOkuda_3.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Akka is reactive and resilient. Those characteristics work really well in enterprise systems and systems integrations. As RMP rises, EIP has been established as one of concrete and modern design options. Although, a number of patterns of RMP and EIP is enormous, the information in Japanese is not enough.
```
Comment (smroy) [inline _candidates_en/YoshitakaOkuda_3.md]: +Akka is reactive and resilient. Those characteristics work really well in enterprise systems and systems integrations. As the use of RMP rises, EIP have been established as concrete and modern design options. Although, a number of patterns of RMP and EIP is enormous, the information in Japanese is not enough.

Comment (smroy) [inline _candidates_en/YoshitakaOkuda_3.md]: "Although, a number of patterns of RMP and EIP is enormous, the information in Japanese is not enough." Does the author mean " Although the number of RMP and EIP are enormous, having the information only accessible in Japanese is not enough"?

**Comment by smroy** [comment]
@yohei1126 I have reviewed the selection. Normally I would be able to edit for grammar etc using the online editor, but I was not able to save any changes that I made, so I left any changes as comments (above).

File: _candidates_en/YoshitakaOkuda_3.md (hunk 2 above)
Comment (yohei1126) [reply _candidates_en/YoshitakaOkuda_3.md]: What he says is the information translated into Japanese is very limited.
I think "having the information only accessible in Japanese is not enough" is good.

**Comment by yohei1126** [comment]
@smroy Thank you. I fixed it.

**Comment by yoskhdia** [comment]
ref: https://github.com/scalajp/2018.scalamatsuri.org/pull/114

File: _candidates_en/YoshitakaOkuda_3.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  - Best Practices
+  - Software Design and Architecture
+  - Microservices
+suggestions:
+  - People who want to read Reactive Messaging Patterns
+  - People who are interested in systems integrations using Akka
+  - People who want to know how to apply Akka
+---
+The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP）was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.
+Akka is reactive and resilient. Those characteristics work really well in enterprise systems and systems integrations. As the use of RMP rises, EIP have been established as concrete and modern design options. Although, a number of patterns of RMP and EIP is enormous, having the information only accessible in Japanese is not enough.
```
Comment (smroy) [inline _candidates_en/YoshitakaOkuda_3.md]: Should read "Although, the number of patterns of RMP and EIP is enormous, having the information only accessible in Japanese is not enough."

**Comment by yohei1126** [comment]
fixed.

**Comment by yohei1126** [comment]
@smroy please merge if you see any problems. Thanks.

**Comment by smroy** [comment]
@yohei1126 All looks good to me. Thanks!

**Pull request merged**